core/services/chainlink/application.go

```diff
@@ -102,6 +105,7 @@ func NewApplication(config *orm.Config, onConnectCallbacks ...func(Application))
 	headTrackables := []strpkg.HeadTrackable{
 		store.TxManager,
 		jobSubscriber,
+		gasUpdater,
```

Comment:
It looks like `OnNewHead` is called synchronously, in turn, for each callback.

Is this a good or bad thing? Not sure...

Reply:
We may want to call gasUpdater before jobSubscriber in this case.

Reply:
Yes absolutely. The downside is that if geth takes a long time to respond we hold up job processing.

Reply:
@coventry what do you think?

Reply:
I'm in favor of the change. If geth is taking a long time to respond, job processing is probably not going to progress very quickly, anyway.

@se3000 @j16r

Reply:
I think it's fine if the first attempt is occasionally delayed on startup due to a low gas price.